### ISSUE TITLE
Satchel of Exotic Mysteries

### ISSUE THREAD

**Issue by wotlkenjoyer** [gamefreedomgit/Frostmourne#1380]
Hello, currently  right after any server restart , Satchel of Exotic Mysteries  appears and keep appearing  for the next 1-2 days, after that  it stop appearing. Usually we have  only 2 days a week   of satchels, and then it disappears, does not matter do queue require a tank/healer or no.

It looks like after some amount of RHC with satchel done, something happens with this reward system, and it never appears any more.  

How to reproduce,  if you try now queue as tank   you most likely will get an instant pop (tank is required)  but you will not have satchels.

**Comment by Manpower99** [comment]
Yup, satchel trigger dissappeared completely. tanks are Qing without benefitting for the past week.

**Comment by zebostplay** [comment]
I'm resto druid, queued many times on RHC and didn't get any satchel even when event was up. all these days, zero satchels. Completely broken even now. i asked gm about this and he told me to report this on bugtracker. i created an account here, but as i see, this problem is reported. Do something with this please.

**Comment by Manpower99** [comment]
Yeah it's been bugged for weeks now.

**Comment by Riztazz** [comment]
Did you guys re-queue as a group while inside LFG maybe? Or did it disappear from the LFG UI while you were in the instance?

**Comment by Riztazz** [comment]
Awesome to hear, that's visual bug mostly and that's what i've been seeing, you're eligable for a reward but it just doesn't display in the UI

**Comment by Riztazz** [comment]
Ok, i've talked with UltraNix and it's a design choice, what matters to you is the time when YOU joined the queue and what was visible for you at that moment, if you were displayed a reward and and one of the roles you queued as matches when group is formed, you will get a reward.
Basically if the text appears when you queue and you are chosen as one of the roles, you will get a reward no matter what UI says

**Comment by Riztazz** [comment]
I'll take that as a no, closing

**Comment by wotlkenjoyer** [comment]
Hello, idk how to prove  this isn't working, but i play this game 18 hours a day nearly, satchels only show up  after server restart, still , working for 12-20 hours and then disappear, disappear forever unless the server gets restart again.  You can queue up at any time of the day when its "bugged" you never see it, and it is not "visual" . 

I get it , it is kinda hard to test without having 100 accounts  for checking the queue 
But i believe  people see the same, here and in discord. Maybe someone gonna say the same

**Comment by RadeghostWM** [comment]
@Riztazz

**Comment by Riztazz** [comment]
Are you doing ascended dungeons or just plain old RDF?

**Comment by wotlkenjoyer** [comment]
> Are you doing ascended dungeons or just plain old RDF?

ascended, heroic , everything, and it does not matter, currently it works like this. 

You load up  the game. Login/Change location with loading (for example portal) 
You get an announcement about  satchels for a tank "Call to arms"
here does not matter what you choose, RHC or ASC 1,2,3  you still gonna get satchel. 
it also does not disappear  if you click different types of queue, everywhere you see satchel, and if you see it it always being awarded.

Problem is, this call to arms works only 1 day max after restart, after that  it disappears, queue can be dead for hours , no tank  and no call to arms, does not matter which dungeon queue, it never appears on your screen, it never appear if you press " I " button. 

Does not matter what population currently, night or morning, 400 or 1400 people, if it is bugged it never appear.

**Comment by wotlkenjoyer** [comment]
I am used to farm satchels whole day if it shows up,  and then open 20-30 in a row  trying to catch mount reward.  I can tell you with 100%  confidence , if you dont see satchel in RDF window - you will never get it.  If it is currently bugged  - you will never see it  even if tanks are required in any of RHC-asc groups

**Comment by wotlkenjoyer** [comment]
> Are you doing ascended dungeons or just plain old RDF?

https://github.com/user-attachments/assets/7a159858-9008-4010-b1b3-ab116a4df8ef

For example right now they are bugged for 2 days already or so, we were farming it for 1 day after those multiple restarts after icc dungeons got released, but now  you never see it even if queue are instant for a tank and people asking for a tank in asc3 in world. idk if it helps.

**Comment by wotlkenjoyer** [comment]
> Are you doing ascended dungeons or just plain old RDF?

i want to mention second time just in case, maybe its a part of a bug and will help.

Exactly what you asked does not make any sense, because  if  you get "Call to arms" message, satchel shows up on every single type of queue, and it does not matter where exactly tank is needed, you can queue for non-ascended RHC  and still get it, even if group arent full.  

But if it is "bugged"    it just never appears anywhere

**Comment by Riztazz** [comment]
Okay, i know what's going on now, now i need to figure out how to go about it:P

**Comment by Riztazz** [comment]
> > Are you doing ascended dungeons or just plain old RDF?
> 
> i want to mention second time just in case, maybe its a part of a bug and will help.
> 
> Exactly what you asked does not make any sense, because if you get "Call to arms" message, satchel shows up on every single type of queue, and it does not matter where exactly tank is needed, you can queue for non-ascended RHC and still get it, even if group arent full.
> 
> But if it is "bugged" it just never appears anywhere

Ascended queues were never counted towards satchel, so yeah, i have reasons for my questions
